package.json

```diff
@@ -22,16 +22,12 @@
     "@babel/core": "^7",
     "@babel/eslint-parser": "^7",
     "@brightspace-ui/stylelint-config": "^0.8",
-    "@open-wc/testing": "^2",
+    "@open-wc/testing": "^3",
     "@web/dev-server": "^0.2",
     "@web/test-runner": "^0.16",
     "@web/test-runner-playwright": "^0.10.0",
     "eslint": "^8",
     "eslint-config-brightspace": "^0.23",
-    "eslint-plugin-html": "^7",
-    "eslint-plugin-import": "^2",
-    "eslint-plugin-lit": "^1",
-    "eslint-plugin-sort-class-members": "^1",
```

Comment:
Don't need these anymore, `eslint-config-brightspace` pulls them in as peer dependencies